pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -32,6 +32,14 @@ type S3RepositoryConfig struct {
 	// TODO: How do we define access? Secrets?
 }
 
+type Workflow string
```

Comment:
```suggestion
// +enum
type Workflow string
```
will trigger codegen to use enum type not string